sdk/storage/azure_storage_blob/CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Release History
 
-## 0.2.0 (Unreleased)
+## 0.2.0 (2025-06-09)
 
 ### Features Added
 
+* Added navigation methods to access sub-clients from existing clients.
```

Comment:
Maybe list out what the methods are? Not sure customers will understand "navigation" methods.